channels/middleware/authorization.go

```diff
@@ -350,30 +343,33 @@ func (am *authorizationMiddleware) authorize(ctx context.Context, session authn.
 
 		switch op {
 		case channels.OpViewChannel:
-			req.Operation = auth.ReadOp
+			req.Operation = auth.ChannelViewOp
 		case channels.OpListUserChannels:
-			req.Operation = auth.ListOp
+			req.Operation = auth.ChannelListOp
 			req.EntityID = auth.AnyIDs
-		case channels.OpUpdateChannel,
-			channels.OpUpdateChannelTags,
-			channels.OpEnableChannel,
-			channels.OpDisableChannel,
-			channels.OpSetParentGroup:
-			req.Operation = auth.UpdateOp
-		case channels.OpDeleteChannel,
-			channels.OpRemoveParentGroup:
-			req.Operation = auth.DeleteOp
+		case channels.OpUpdateChannel:
+			req.Operation = auth.ChannelUpdateOp
+		case channels.OpUpdateChannelTags:
+			req.Operation = auth.ChannelUpdateTagsOp
+		case channels.OpEnableChannel:
+			req.Operation = auth.ChannelEnableOp
+		case channels.OpDisableChannel:
+			req.Operation = auth.ChannelDisableOp
+		case channels.OpSetParentGroup:
+			req.Operation = auth.ChannelSetParentGroupOp
+		case channels.OpDeleteChannel:
+			req.Operation = auth.ChannelDeleteOp
+		case channels.OpRemoveParentGroup:
+			req.Operation = auth.ChannelRemoveParentGroupOp
 		case channels.OpConnectClient:
-			req.Operation = auth.CreateOp
+			req.Operation = auth.ChannelConnectClientOp
 		case channels.OpDisconnectClient:
-			req.Operation = auth.DeleteOp
-		case domains.OpCreateDomainChannels,
-			domains.OpListDomainChannels:
-			if op == domains.OpCreateDomainChannels {
-				req.Operation = auth.CreateOp
-			} else {
-				req.Operation = auth.ListOp
-			}
+			req.Operation = auth.ChannelDisconnectClientOp
```

Comment:
Let's stay consistent in naming, compare this to [ClientConnectToChannelOp](https://github.com/absmach/supermq/pull/3295/changes#diff-b78fc26b2e6dbc94bc9d8a3dba809777dbd4891a71147cc00536e5cc4ea53adbR300).